student-store-express-api/routes/products.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const { storage } = require('../data/storage');
+
+router.get('/', (req, res) => {
+  const items = storage.get('products').value();
```

Comment:
what is .value supposed to do? I would have expected .get('products') to return an array of products

Reply:
.get('product') was  throwing an error but for some reason adding the .value helped solved it

Reply:
> .get('product') was throwing an error but for some reason adding the .value helped solved it

Can you share what the error is?